login.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/*
+*  Pi-hole: A black hole for Internet advertisements
+*  (c) 2017 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license.
+*/
+
+require 'scripts/pi-hole/php/password.php';
+
+// Go directly to index, if authenticated.
+if ($_SESSION['auth']) {
+    header('Location: index.php');
+    exit;
+}
+
+require 'scripts/pi-hole/php/header.php';
+?>
+<body class="hold-transition layout-boxed login-page">
+<div class="box login-box">
+    <section style="padding: 15px;">
+        <div class="login-logo">
+            <div class="text-center">
+                <img src="img/logo.svg" alt="Pi-hole logo" class="loginpage-logo">
+            </div>
+            <div class="panel-title text-center"><span class="logo-lg" style="font-size: 25px;">Pi-<b>hole</b></span></div>
+        </div>
+        <!-- /.login-logo -->
+
+        <div class="card">
+            <div class="card-body login-card-body">
+                <p class="login-box-msg">Sign in to start your session</p>
```

Comment:
Do we need this text at all? If so, maybe 
```suggestion
                <p class="login-box-msg">Sign in to access Pi-hole's dashboard</p>
```

Reply:
I don't have a strong opinion about this, but I would follow one of this ideas:
1. remove the text;
2. use "Sign in to access the dashboard"
   I don't think we need to use "Pi-hole" again. The name and logo are is already above this line.

Reply:
Then I would remove the text. What else should happen after you clicked "log in" on a login page other than being logged in?

Reply:
I agree.